src/Microsoft.ML.TensorFlow/TensorflowCatalog.cs

```diff
@@ -11,6 +11,26 @@ namespace Microsoft.ML
     /// <include file='doc.xml' path='doc/members/member[@name="TensorflowTransformer"]/*' />
     public static class TensorflowCatalog
     {
+        /// <summary>
+        /// Scores a dataset using a pre-traiend TensorFlow model located in <paramref name="modelLocation"/>.
+        /// </summary>
+        /// <param name="catalog">The transform's catalog.</param>
+        /// <param name="modelLocation">Location of the TensorFlow model.</param>
+        /// <param name="inputColumnName"> The name of the model inputs.</param>
+        /// <param name="outputColumnName">The name of the requested model outputs.</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        /// [!code-csharp[ScoreTensorFlowModel](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlowTransform.cs)]
```

Comment:
>TensorFlowTransform [](start = 108, length = 19)

Zeeshan A is trying to rename this file, so be careful
[https://github.com/dotnet/machinelearning/pull/2429](https://github.com/dotnet/machinelearning/pull/2429)

Reply:
please merge with master 
[https://github.com/dotnet/machinelearning/pull/2429](https://github.com/dotnet/machinelearning/pull/2429) got merge and this file got moved.

---
In reply to: [254137918](https://github.com/dotnet/machinelearning/pull/2367#discussion_r254137918) [](ancestors = 254137918)

Reply:
Just rebased again, should be good now.

---
In reply to: [254514215](https://github.com/dotnet/machinelearning/pull/2367#discussion_r254514215) [](ancestors = 254514215,254137918)

Reply:
Did you push your changes? because I still see `Dynamic/TensorFlowTransform.cs` instead of `TensorFlow/ImageClassification.cs`

---
In reply to: [254515347](https://github.com/dotnet/machinelearning/pull/2367#discussion_r254515347) [](ancestors = 254515347,254514215,254137918)

Reply:
I am sorry I think I did not rebase earlier. I now rebased and pushed again and I see Zeeshan's change now.

Reply:
I still see in line 24 link to         `/// [!code-csharp[ScoreTensorFlowModel](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlowTransform.cs)]`

And can you please stop using your rebase magic? Codeflow continue to stay on same iteration, and it get's hard to track which files get changed and which are not?


---
In reply to: [254547579](https://github.com/dotnet/machinelearning/pull/2367#discussion_r254547579) [](ancestors = 254547579)

Reply:
Ah now I understood what you meant! the links will be broken sorry!! Fixing right away

Reply:
Ok now I think I did what you were saying from the beginning, I didn't understand you were talking about the link. Just pushed a new iteration.